README.md

```diff
@@ -437,6 +447,14 @@ A property attribute MUST be one of these:
         <td>Yes</td>
         <td>No (<code>false</code>)</td>
     </tr>
+    <tr>
+        <td>$retained</td>
+        <td>Device → Controller</td>
+        <td>Specifies whether the property is retained (<code>true</code>) or non-retained (<code>false</code>). Publishing to a non-retained property topic MUST always happen with the MQTT 'retain' flag off.</td>
```

Comment:
Shouldn't the last sentence be "Publishing to a non-retained property topic MUST always happen with the MQTT 'retain' flag ¡false'."? IMHO, the 'off' word could be confusing.

Reply:
"with the MQTT 'retain' flag off" means without the `retain` flag, not with the flag set to 'off', so I'd say it's fine.